helm/main.go

```diff
@@ -268,7 +392,8 @@ func (h *Helm) doesChartExistOnRepo(
 
 		if exc == "0" {
 			//Chart exists
-			return true, nil
+			return true, fmt.Errorf("Debug")
```

Comment:
```suggestion
			return true,  nil
```

Should return nil here. 
Returning "Debug" would fail with the original `HelmPackagePushWithExistingChart` test function, but somehow passes with the new test.
Original test: https://github.com/puzzle/dagger-module-helm/blob/main/tests/main.go#L105

Reply:
This has been changed back to `nil`.